p2p/host/eventbus/basic_metrics.go

```diff
@@ -0,0 +1,139 @@
+package eventbus
+
+import (
+	"reflect"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const metricNamespace = "libp2p_eventbus"
+
+var (
+	eventsEmitted = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricNamespace,
+			Name:      "events_emitted_total",
+			Help:      "Events Emitted",
+		},
+		[]string{"event"},
+	)
+	notificationTime = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricNamespace,
+			Name:      "notification_time_microseconds",
```

Comment:
There's a convention in Prometheus to use SI units: https://prometheus.io/docs/practices/naming/. So this should be `notification_time_seconds`.

Reply:
Thinking about this a bit more, this measurement doesn't seem as useful any more:
* If all channels don't block, this takes basically no time at all, we're just adding an event to every channel.
* If a channel blocks because the subscriber is not reading from the channel, it will block indefinitely, and then this event won't be emitted.

I suggest remote it.

Reply:
fixed